static/app/views/alerts/rules/metric/details/sidebar.tsx

```diff
@@ -260,6 +263,40 @@ export function MetricDetailsSidebar({
               teamActor ? <ActorAvatar actor={teamActor} size={24} /> : t('Unassigned')
             }
           />
+          {rule.detectionType === 'dynamic' && (
+            <KeyValueTableRow
+              keyName={t('Sensitivity')}
+              value={
+                rule.sensitivity
+                  ? rule.sensitivity.charAt(0).toUpperCase() + rule.sensitivity.slice(1)
+                  : ''
+              } // NOTE: if the rule is dynamic, then there must be a sensitivity
+            />
+          )}
+          {rule.detectionType === 'dynamic' && (
+            <KeyValueTableRow
+              keyName={t('Seasonality')}
+              value={
+                rule.seasonality
+                  ? rule.seasonality.charAt(0).toUpperCase() + rule.seasonality.slice(1)
+                  : ''
+              } // NOTE: if the rule is dynamic, then there must be a seasonality
+            />
+          )}
```

Comment:
Do we still need this given we're not exposing seasonality anymore?

Reply:
Oops